SeldonIO/seldon-core
# 57 wrapping auto docs

Related to issue #57 

To test: 
- build core-python-wrapper
- use core-python-wrapper to wrap an example microservice
- build the wrapped microservice
- The build folder should contain a readme.md file and docker inspect <image_name> should contain a list of labels reflecting the argument that were used when wrapping
